system/Debug/Toolbar.php

```diff
@@ -365,7 +365,7 @@ public function prepare(?RequestInterface $request = null, ?ResponseInterface $r
                 return;
             }
 
-            $toolbar = Services::toolbar(config(self::class));
```

Comment:
`self::class` is `CodeIgniter\Debug\Toolbar`, and the basename is `Toolbar`.
So Factories will load `Config\Toolbar` (preferApp) now.

Reply:
But with #7733, Factories will try to instantiate `CodeIgniter\Debug\Toolbar`, 
and it causes:
```
ArgumentCountError
Too few arguments to function CodeIgniter\Debug\Toolbar::__construct(), 0 passed in 
/.../CodeIgniter4/system/Config/Factories.php on line 183 and exactly 1 expected
```

Reply:
I'm not sure; that's an odd one. Maybe some kind of autoloading loop?